src/components/SidebarNav/SidebarNav.jsx

```diff
@@ -0,0 +1,30 @@
+import "../../css/main.css";
+import "./SidebarNav.css";
+import { Link, useLocation } from "react-router-dom";
+import { SidebarNavData } from "./SidebarNavData";
+
+export const SidebarNav = () => {
+  const location = useLocation();
+  return (
+    <aside className="container-sidebar">
+      <ul className="list-style-none">
+        {SidebarNavData.map(({ _id, name, icon, link }) => {
+          return (
+            <Link
```

Comment:
suggestion : you can use the navlink, as it has isActive attribute

Reply:
Done, used NavLink instead of Link